pages/download.html

```diff
@@ -49,11 +49,11 @@ <h3>1.4.5</h3>
 <li><a href="https://www.asp.net/ajaxLibrary/CDN.ashx#jQuery_Mobile_Releases_on_the_CDN_4">Microsoft CDN &#8211; jQuery Mobile</a></li>
 </ul>
 
-<hr>
+<hr />
```

Comment:
This is HTML; why would we self-close when it has no meaning in HTML?

Reply:
I don't really have a preference, but many tags were already self-closing so this at least makes it consistent.

Reply:
Fair enough.